turn egauge notebook into script

matthew-schultz opened the pull request:
Took the egauge python notebook and created a python script.

Also created a test shell script that with arguments that send a request to the running egauge sensor with the id 725. 

There is also a commented out line for an offline egauge sensor to test the python script's response to error conditions (if it can't connect to the egauge sensor, etc). You may need to change the shell scripts permissions using ```chmod u+x test_egauge_api_01.sh``` from the shell while you are in the /sensors directory before running the script.

matthew-schultz commented:
@ryantanaka helped me get rid of the merge commit, so it should be good now.

carlosparadis commented on egauge/egauge_api.py:
```diff
@@ -0,0 +1,63 @@
+if __name__ == "__main__":
+    script, start, end, egauge_id, file_path = sys.argv
+
+    egauge_request(start, end, egauge_id, file_path)
```
Flow my comments starting from here: Jump to line 13.

carlosparadis commented on egauge/egauge_api.py:
```diff
@@ -0,0 +1,63 @@
+import traceback
+
+def egauge_request(start_timestamp, end_timestamp, sensor_id, output_file):
+    host = set_egauge_host(sensor_id)
```
Step 2) Jump to line 52.

carlosparadis commented on egauge/egauge_api.py:
```diff
@@ -0,0 +1,63 @@
+    + minutes + '&' \
+    + output_csv + '&' \
+    + delta_compression
+    return host
```
Step 3) Jump to line 63. Do you still remember Step 2? See comment on the PR why I had you do this exercise.

matthew-schultz commented:
@carlosparadis I updated egauge_api.py to run within main(), cleaned up the code, and moved all my files to a /script subdirectory.

How does it look so far?

Thank you!

carlosparadis commented:
@matthew-schultz sorry it took me so long to go over this -- it looks fine, thanks for understanding!
